src/parser/FStar.Parser.Lexhelp.fs

```diff
@@ -173,6 +173,7 @@ let keywords =
     ALWAYS, "let"        ,LET(false);
     ALWAYS, "logic"      ,LOGIC;
     ALWAYS, "match"      ,MATCH;
+    ALWAYS, "as"         ,AS;
```

Comment:
super tiny nit, sorry. But this list of keywords was originally meant to be in alphabetical order. I see there are some deviations from that already, but may be worth restoring at some point.